landing-page/src/MobileView.tsx

```
@@ -0,0 +1,680 @@
+              <img src={madanilogo} alt="madani" width="240px" height="92px" />
+            </Stack>
+            <Stack alignItems="center">
+              <Typography fontSize="32px" fontWeight="600" color="#00254D">{madaniInfo.mobileTitle1}</Typography>
+              <Typography fontSize="32px" fontWeight="600" color="#00254D">{madaniInfo.mobileTitle2}</Typography>
+            </Stack>
+            <Stack gap="16px" alignItems="center">
+              <Typography fontSize="16px" fontWeight="600" color="#11181C" sx={{textIndent: "2em", textAlign: "center"}}>{madaniInfo.subtitle}</Typography>
+              <Stack>
+                <Stack gap="4px">
+                  {madaniInfo.description.map(point => (
+                    <Stack flexDirection="row" gap="8px" alignItems="center">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)